include/swift/Reflection/RuntimeInternals.h

```diff
@@ -94,28 +94,34 @@ struct StackAllocator {
 };
 
 template <typename Runtime>
-struct ActiveTaskStatus {
+struct ActiveTaskStatusWithEscalation {
+  uint32_t Flags;
+  uint32_t DrainLock[(sizeof(typename Runtime::StoredPointer) == 8) ? 1 : 2];
   typename Runtime::StoredPointer Record;
-  typename Runtime::StoredSize Flags;
 };
 
 template <typename Runtime>
+struct ActiveTaskStatusWithoutEscalation {
+  uint32_t Flags[sizeof(typename Runtime::StoredPointer) == 8 ? 2 : 1];
```

Comment:
This also looks wrong.

Reply:
Same hack logic as above: 

32 bit without priority escalation:

```
uint32_t flags
typename Runtime::StoredPointer statusRecords
```
64 bit without priority escalation:
```
uint32_t flags
uint32_t unused
typename Runtime::StoredPointer statusRecords
```